package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "mpd-parser",
-  "version": "0.7.0",
+  "version": "0.8.0-0",
```

Comment:
I'm guessing we should revert this version bump here.

Reply:
probably fine to leave it, as we'll just do a minor release (which will remove the -0) from it.